sentry_sdk/utils.py

```diff
@@ -665,6 +665,11 @@ def get_errno(exc_value):
     # type: (BaseException) -> Optional[Any]
     return getattr(exc_value, "errno", None)
 
+def get_error_message(exc_value):
+    # type: (BaseException) -> str
+    return (getattr(exc_value, "message", "") 
+            or getattr(exc_value, "detail", "") 
```

Comment:
Let's be safe, just in case:
```suggestion
            or safe_str(getattr(exc_value, "detail", ""))
```